packages/core/crates/core-crypto/src/ec_groups.rs

```diff
@@ -181,11 +181,13 @@ mod tests {
         assert_eq!(res, key.as_ref());
     }
 
+    #[ignore]
     #[parameterized(nodes = {4, 3, 2, 1})]
     fn test_secp256k1_suite(nodes: usize) {
         generic_sphinx_suite_test::<Secp256k1Suite>(nodes)
     }
 
+    #[ignore]
```

Comment:
Why the ignore?

Reply:
We do not use these instantiations anywhere, only the X25619 one